identify/extensions.py

```diff
@@ -117,6 +117,7 @@
     'whl': {'binary', 'wheel', 'zip'},
     'woff': {'binary', 'woff'},
     'woff2': {'binary', 'woff2'},
+    'wsgi', {'text', 'wsgi', 'python'},
```

Comment:
derp, nice comma

Reply:
lol gdi

Reply:
that's what I get by contributing from github ui and not using `pre-commit` :'(